src/seedu/addressbook/ui/TextUi.java

```diff
@@ -16,6 +10,8 @@
 import seedu.addressbook.commands.CommandResult;
 import seedu.addressbook.data.person.ReadOnlyPerson;
 
+import static seedu.addressbook.common.Messages.*;
```

Comment:
[Coding standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html) violation! 
Import statements should be explicit instead of being grouped together.

There should be a document in the developer guide on how to set up IntelliJ so that it does not automatically group the imports together.